README.md

```diff
@@ -2,24 +2,26 @@
 
 You can use this Ruby client to integrate your service with [GOV.UK Registers](https://registers.cloudapps.digital/).
 
-Registers are authoritative lists of information, and the data is owned by 'custodians' inside departments and services. For example, the [country register](https://country.register.gov.uk/) is maintained by a custodian in the Foreign and Commonwealth Office (FCO). 
+Registers are authoritative lists of information. The data is owned by [custodians](https://www.gov.uk/government/collections/registers-guidance#creating-and-managing-registers) inside departments and services. For example, the [Country register](https://country.register.gov.uk/) is maintained by a custodian in the Foreign and Commonwealth Office (FCO). 
 
 ## Table of Contents
 
 * [Installation](#installation)
-* [Getting started](#getting-started)
+* [Get started](#get-started)
 * [Items, entries and records](#items-entries-and-records)
 * [Collections](#collections)
-* [`RegisterClient`](#registerclient) 
+* [The `RegisterClient` class](#registerclient) 
```

Comment:
I would expect this text to be the same in linked the header.

Aside, perhaps it's just me but I find the items of the ToC to be a bit off (as in not of the same type). "Installation" and "Get started" are of one type, "Items, entries and records" could be of the same type, "Collections" not sure but "The `RegisterClient` class" is definitely of a different type or, it sets different expectations of what I will find there.

What do you think?

Reply:
@arnau Whoops, that first point was just my mistake. Thanks for spotting.

The second point I agree on totally, and was actually going to open a separate PR for that. I think the first two parts of the material are effectively "instructions" whereas the rest of it is "reference material". If we're going to keep to this format, which was one of the things I was hoping to discuss with you and @karlbaker02 on Monday, then I think a natural change would be to separate the two sections as such. 

I'm trying to only open small PRs to make small changes rather than do the whole thing at once, but if you think it would be easier to do the latter then do let me know; I was trying to make things a bit easier to manage but if it's having the opposite effect don't hesitate to tell me.

Reply:
Makes sense 👌